ISS/views/forum.py

```diff
@@ -444,8 +444,7 @@ def usercp(request):
     threads = (Thread.objects.all()
         .filter(
             threadflag__poster_id=request.user.id,
-            threadflag__subscribed=True,
-            last_update__gt=F('threadflag__last_read_date'))
+            threadflag__subscribed=True)
```

Comment:
Lol, we talked about this. You, me, and lala couldn't agree on how it should work so I'm going to say let's take this out of this PR. If you want to run a poll on it on the site feel free, but this was vBul's pattern and I think it works. I'll defer to that unless there's good evidence that _most_ people want something different.

Reply:
Removed